lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -158,7 +157,7 @@
                                     [tooltip]="getCellTooltip(row, col)">
                                 </adf-location-cell>
                             </div>
-                            <div *ngSwitchCase="'fileSize'" class="adf-cell-value"
+                            <div *ngSwitchCase="'fileSize'" [attr.tabindex]="data.getValue(row, col)? 0 : -1" class="adf-cell-value"
```

Comment:
what is the purpose of this code? skipping the empty cell?

Reply:
for folders we don't have size  value so we skip it.

Reply:
Is it ok to skip it though? Won't this be potentially confusing for screen reader users?

Reply:
in what way?